projects/plugins/jetpack/class.jetpack.php

```diff
@@ -3913,6 +3913,23 @@ public function upload_handler( $update_media_item = false ) {
 			return new WP_Error( 'unknown_token', 'Unknown Jetpack token', 403 );
 		}
 
+		/**
+		 * Optionally block uploads processed through jetpack's upload_handler().
```

Comment:
```suggestion
		 * Optionally block uploads processed through Jetpack's upload_handler().
```

Since this will end up on it's own hook page on developer.jetpack.com, I'd suggest changing jetpack to Jetpack here.

Reply:
Fixed in 7c6ac6e381